docs/coverage.md

```diff
@@ -0,0 +1,78 @@
+# Rule coverage
+
+ATT&CK navigator layer files are generated when a package is built with `make release` or `python -m detection-rules`.
+This also means they can be downloaded from all successful builds. 
+
+These files can be used to pass to a custom navigator
+session. For convenience, the links are generated below. You can also include multiple across tabs in a single session, 
+though it is not advisable to upload _all_ of them as it will likely overload your browsers resources.
+
+## Current rule coverage
+
+The source files for these links are regenerated with every successful merge to main. These represent coverage from the 
+state of rules in the `main` branch.
+
+
+| Navigator links |
+|-----------------|
+|[all.json](https://mitre-attack.github.io/attack-navigator/#layerURL=https%3A%2F%2Fgist.githubusercontent.com%2Fbrokensound77%2F1a3f65224822a30a8228a8ed20289a89%2Fraw%2F5db621844a142bdb8f537449ceb8d3a817a34586%2Fall.json&leave_site_dialog=false&tabs=false)|
+|[indexes-auditbeat-*.json](https://mitre-attack.github.io/attack-navigator/#layerURL=https%3A%2F%2Fgist.githubusercontent.com%2Fbrokensound77%2F1a3f65224822a30a8228a8ed20289a89%2Fraw%2F30342860fa830b90c025779770b9ca7b13192061%2Findexes-auditbeat-%2A.json&leave_site_dialog=false&tabs=false)|
```

Comment:
some of these are throwing parsing errors from navigator. I will look into it.

<details>

error
```
main-es2018.84172b115fe9ba0742ee.js:formatted:130688 TypeError: Cannot read properties of null (reading 'name')
    at gm.deSerializeDomainVersionID (main-es2018.84172b115fe9ba0742ee.js:formatted:128635:31)
    at Object.next (main-es2018.84172b115fe9ba0742ee.js:formatted:130679:39)
    at u.__tryOrUnsub (main-es2018.84172b115fe9ba0742ee.js:formatted:24047:23)
    at u.next (main-es2018.84172b115fe9ba0742ee.js:formatted:24010:157)
    at l._next (main-es2018.84172b115fe9ba0742ee.js:formatted:23970:34)
    at l.next (main-es2018.84172b115fe9ba0742ee.js:formatted:23955:40)
    at s._next (main-es2018.84172b115fe9ba0742ee.js:formatted:84031:34)
    at s.next (main-es2018.84172b115fe9ba0742ee.js:formatted:23955:40)
    at s._next (main-es2018.84172b115fe9ba0742ee.js:formatted:89949:39)
    at s.next (main-es2018.84172b115fe9ba0742ee.js:formatted:23955:40)
next @ main-es2018.84172b115fe9ba0742ee.js:formatted:130688
```

source
```javascript
async loadLayerFromURL(e, t, n=!1) {
                return new Promise(async(r,i)=>{
                    let o = this.http.get(e).subscribe({
                        next: async i=>{
                            let o = this.viewModelsService.newViewModel("loading layer...", void 0);
                            try {
                                if (o.deSerializeDomainVersionID(i),
                                !this.dataService.getDomain(o.domainVersionID))
                                    throw {
                                        message: "Error: '" + o.domain + "' (" + o.version + ") is an invalid domain."
                                    };
                                await this.layerUpgrade(o, i, t, n),
                                console.log("loaded layer from", e),
                                r(null)
                            } catch (s) {
                                console.error(s),
                                this.viewModelsService.destroyViewModel(o),
                                alert("ERROR parsing layer from " + e + ", check the javascript console for more information."),
                                r(null)
                            }
```

</details>